projects/mercury/src/routes/WorkspaceRoutes.js

```diff
@@ -107,7 +119,7 @@ const WorkspaceRoutes = () => {
                 render={() => (isAdmin(currentUser) && (<UserRolesPage />))}
             />
 
-            <Redirect to="/workspaces" />
+            <Redirect to="/metadata-views" />
```

Comment:
is this correct?

Reply:
oups, no, included by mistake, I'll revert it.